apps/include/apps.h

```diff
@@ -271,6 +271,7 @@ typedef struct app_http_tls_info_st {
 } APP_HTTP_TLS_INFO;
 BIO *app_http_tls_cb(BIO *hbio, /* APP_HTTP_TLS_INFO */ void *arg,
                      int connect, int detail);
+void APP_HTTP_TLS_INFO_free(APP_HTTP_TLS_INFO *info);
```

Comment:
Why the uppercase `APP_` prefix?

Reply:
The structure is named like that.

Reply:
then `app_http_tls_cb` looks misnamed when the two are adjacent.
shrug.

Reply:
I would accept renaming the function to lowercase, it works for me as well.

Reply:
For any `TYPE`, its destructor is called `TYPE_free()`, therefore it was natural to use the name `APP_HTTP_TLS_INFO_free`.
The function `app_http_tls_cb()` is related to `APP_HTTP_TLS_INFO`, but is of more general nature:
it's a TLS callback function for HTTP client used by apps. 
So I see no reason to use other names.